docs/contributing/coding-standards/js.md

```diff
@@ -85,35 +87,37 @@ Use `// FIXME:` to annotate problems.
 
 Use `// TODO:` to annotate solutions to problems.
 
-## Constructors and methods
+## Classes and methods
 
-Use the prototype design pattern to structure your code.
+Use the class design pattern to structure your code.
 
-Create a constructor and define any variables that the object needs.
+Create a class and define the methods you need.
 
 ```mjs
-function Example ($module) {
+class Example {
   // Code goes here
 }
 ```
 
-Assign methods to the prototype object. Do not overwrite the prototype with a new object as this makes inheritance impossible.
+Add methods to the class. Do not overwrite the prototype with a new object as this makes inheritance impossible.
```

Comment:
I _think_ we can delete this bit as it's no longer relevant?

```suggestion
Add methods to the class.
```

Reply:
Left it in as a technicality (still possible I guess?) but also glad to remove!

Reply:
Removed it ✅